english/spooky-house/spooky-house.md

```diff
@@ -0,0 +1,14 @@
+as you walk across a large field, you notice a house near a group of trees.  
+Your intuition tells you to stay away yet you find yourself walking towards it.  
+You make it to the doorstep, push the door open and start walking inside the house.  
+To the right of the entrance is a large kitchen area, filled with cob-webs.  
+To the left, in your horror, you notice oddly shaped footprints on the floor.  
+You say to yourself "these footprints are fresh, and they smell like marshmallows".  
+You sense something breathing hard behind you.  
+You say to yourself - "It can't be the marshmallow man, he only exists in movies".  
+Low and behold.  It is the marshmallow man. He jumps on you, tries to suffocate you with his puffy marshmallowy body.
+In your defence, you start eating away.  He retreats.  Runs out the entrance door and stops for a moment.  
+He turns his head and says to you.
+"This ain't over, I'll be back with all of my marshmallow friends to finish the job".  
+After catching your breath, you take a moment to think about the series of events that has just happened.
+You swallow the last pieces of marshmallow in your mouth and say to yourself - "you bring it marshmallow man, you bring it".
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.